debian/cozy-standalone.postrm

```diff
@@ -44,6 +44,12 @@ case "$1" in
 		[ -d /usr/local/var/log/cozy ] && echo "Delete /usr/local/var/log/cozy directory" && rm -rf /usr/local/var/log/cozy
 		[ -d /usr/local/cozy ] && echo "Delete /usr/local/cozy directory" && rm -rf /usr/local/cozy
 		[ -d /var/www/cozy-letsencrypt ] && echo "Delete /var/www/cozy-letsencrypt directory" && rm -rf /var/www/cozy-letsencrypt
+		
+		echo "Delete Cozy's UNIX users"
+		users=`cat /etc/passwd | cut -d":" -f1 | grep "^cozy\-"`
+		for user in $users; do
+			userdel $user
```

Comment:
@babolivier [the manpage](http://manpages.ubuntu.com/manpages/trusty/man8/userdel.8.html) says:

>  userdel is a low level utility for removing users. On Debian, administrators should usually use deluser(8) instead.

And shouldn't we delete the home directory as well?